src/eventsService.ts

```diff
@@ -0,0 +1,28 @@
+const GET_UPCOMING_EVENTS_URL =
+	'https://2sif0durcj.execute-api.eu-west-1.amazonaws.com/dev/get-upcoming-events';
+
+let cachedUpcomingEvents: any[];
```

Comment:
Would it be nice to use `sessionStorage` or `localStorage` for this?

Reply:
It was on purpose to retrieve the info on every reload, just not twice on every reload (once for the navbar and again for the events page). If it would be stored even in sessionStorage, it might be cached too long.

Reply:
Hmm okay, valid point!